thavalon-server/src/connections/errors.rs

```diff
@@ -24,6 +25,8 @@ enum ErrorCode {
     InvalidLogin,
     MissingHeader,
     InvalidAccountVerification,
+    AccountNotVerified,
```

Comment:
For the discussion about error handling in #54, I think we could match on these codes since they're already getting sent back